bin/build-release-ccd-definition.sh

```diff
@@ -12,7 +12,7 @@ fi
 if [ ${environment} == "prod" ]; then
   excludedFilenamePatterns="-e UserProfile.json,*-nonprod.json"
 else
-  excludedFilenamePatterns="-e UserProfile.json"
+  excludedFilenamePatterns="-e UserProfile.json,*-prod.json"
```

Comment:
@joez-f will you need this to? Happy to make a separate PR

Reply:
Can just do
```
git checkout origin/FPLA-1990-View-Application-In-Open-State bin/import-ccd-definition.sh
git checkout origin/FPLA-1990-View-Application-In-Open-State bin/build-release-ccd-definition.sh
```
if you can't be bothered to